scripts/Muon/GUI/ElementalAnalysis/peak_data.json

```diff
@@ -3,2370 +3,2463 @@
+      "K( 2p->1s)": 4563.2,
+      "K(2p ->1s)": 4574.2,
+      "L(3d->2p)": 1529.5,
+      "L(3d->2p) ": 1539.1,
+      "L(3d->2p)  ": 1550.8,
+      "L(3d->2p)   ": 1621.4,
+      "L(3d->2p)    ": 1637.0,
+      "M(4f->3d)": 550.5,
+      "M(4f->3d) ": 562.7,
+      "N(5g->4f)": 253.4,
+      "N(5g->4f) ": 255.7,
+      "O(6h->5g)": 137.8
+    },
+    "Secondary": {
+      "K(2p ->1s)": 4544.3,
+      "K( 2p->1s)": 4588.8,
+      "L(3d->2p)": 1573.3,
+      "L(3d->2p) ": 1597.6,
+      "M(5f->3d)": 802.8,
+      "M(5f->3d) ": 816.4,
+      "M(6f->3d)": 939.1,
+      "M(6f->3d) ": 953.2,
+      "N(6g->4f)": 390.1,
+      "N(6g->4f) ": 392.8,
+      "N(7g->4f)": 472.5,
+      "N(7g->4f) ": 475.4,
+      "O(7h->5g)": 220.1
     }
   },
   "F": {
     "Z": 9,
     "A": 19,
     "Primary": {
-      "K(2->1)": 168.1
+      "K(2p->1s)": 168.1,
+      "L(3d->2p)": 31.6,
+      "M(4f->3d)": 11.1
     },
     "Secondary": {
-      "K(3->1)": 199.7,
-      "K(4->1)": 210.8,
-      "K(5->1)": 216.3,
-      "L(3->2)": 31.6,
-      "L(4->2)": 42.7,
-      "M(4->3)": 11.1
+      "K(3p->1s)": 199.7,
+      "K(4p->1s)": 210.8,
+      "K(5p->1s)": 216.3,
+      "L(4d->2p)": 42.7
     }
   },
   "Fe": {
     "Z": 26,
     "A": 55.85,
     "Primary": {
-      "K(3->1)": 1257.9,
-      "L(3->2)": 265.7,
-      "L(4->2)": 269.4,
-      "M(4->3)": 92.6,
-      "5->4": 42.1
-    },
-    "Secondary": {
-      "K(2->1)": 1253.7,
-      "K(4->1)": 1521.1,
-      "K(5->1)": 1523.5,
-      "K(6->1)": 1616.7,
-      "K(7->1)": 1659.7,
-      "K(8->1)": 1683.3,
-      "K(9->1)": 1697.7,
-      "K(10->1)": 1706.4,
-      "L(5->2)": 358.2,
-      "L(6->2)": 361.9,
-      "L(7->2)": 400.8,
-      "L(8->2)": 404.7,
-      "L(9->2)": 424.3,
-      "L(10->2)": 428.1,
-      "L(11->2)": 438.3,
-      "L(12->2)": 442.2,
-      "M(5->3)": 134.9,
-      "M(6->3)": 157.8,
-      "M(7->3)": 172.3,
-      "6->4": 65.6,
-      "7->4": 79.6
+      "K(2p1/2->1s1/2)": 1253.7,
+      "K(2p3/2->1s1/2)": 1257.9,
+      "L(3d5/2->2p3/2)": 265.7,
+      "L(3d1/2->2p3/2)": 269.4,
+      "M(4f->3d)": 92.6,
+      "N(5g->4f)": 42.1
+    },
+    "Secondary": {
+      "K(3p->1s)": 1521.1,
+      "K(3p->1s) ": 1523.5,
+      "K(4p->1s)": 1616.7,
+      "K(5p->1s)": 1659.7,
+      "K(6p->1s)": 1683.3,
+      "K(7p->1s)": 1697.7,
+      "K(8p->1s)": 1706.4,
+      "L(4d->2p)": 358.2,
+      "L(4d->2p) ": 361.9,
+      "L(5d->2p)": 400.8,
+      "L(5d->2p) ": 404.7,
+      "L(6d->2p)": 424.3,
+      "L(6d->2p) ": 428.1,
+      "L(7d->2p)": 438.3,
+      "L(8d->2p)": 442.2,
+      "M(5f->3d)": 134.9,
+      "M(6f->3d)": 157.8,
+      "M(7f->3d)": 172.3,
+      "N(6g->4f)": 65.6,
+      "N(7g->4f)": 79.6
     },
     "Gammas": {
-      "g54Mn25 (1)": 54.5,
-      "g54Mn25 (2)": 156.2,
-      "g54Mn25 (3)": 212,
-      "g54Mn25 (4)": 251.6,
-      "g54Mn25 (5)": 376.9,
-      "g55Mn25": 126
+      "g54Mn (100)": 54.5,
+      "g54Mn(100)": 156.2,
+      " g54Mn (100)": 212,
+      "  g54Mn(100)": 251.6,
```

Comment:
Are the whitespace differences here intentional?  They are carried through into the GUI and look messy.